test/test_group.py

```diff
@@ -48,8 +48,8 @@ def test_group_2rect(self):
         r1 = rects[1]
         self.assertEqual(r1.implicit_x, 0)
         self.assertEqual(r1.implicit_y, 0)
-        self.assertEqual(r1.implicit_width, 200)
-        self.assertEqual(r1.implicit_height, 200)
+        self.assertEqual(round(r1.implicit_width,12), 200)
+        self.assertEqual(round(r1.implicit_height,12), 200)

```

Comment:
These would need to change to assertAlmostEqual() rather than being rounded here.

Reply:
Agreed.